CMakePresets.json

```diff
@@ -13,8 +13,6 @@
         "BUILD_SHARED_LIBS": "ON",
         "ENABLE_WEBSERVER": "OFF",
         "SELF_BUILT_AUBIO": "ALWAYS",
-        "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/${presetName}-install",
-        "CMAKE_TOOLCHAIN_FILE": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
```

Comment:
Please keep these lines since else it'll break visual studio users. You can always overwrite these at the actual cmake script.